### ISSUE TITLE
Update User Guide for v3.4.0

### ISSUE THREAD

**Issue by john-shepherdson** [cessda/cessda.cdc.versions#602]
Hi John,

We have had a look at the CDC User Guide and think that it should be updated with the new release to include the new Keyword filtering functionality as well as some other smaller updates. Is text for the User Guide available somewhere? And how do I go about editing it?

We also noticed that the About page should be updated as it refers to version 3.2.0 released on 2022-12-08. Is it possible to edit this page too somehow?

Kristina

**Comment by john-shepherdson** [comment]
Thank you! I will look through the User Guide and About section and see which changes should be made. If we want to replace some of the pictures or include new ones should this be in the GDoc then?

Kristina

**Comment by john-shepherdson** [comment]
Yes, please edit the GDoc if you want to modify the pictures in any way.

John

**Comment by john-shepherdson** [comment]
@KristinaS4 Please use Suggesting mode (rather than Editing mode), so that I can see where you have made changes, and acknowledge them as I update the online documentation. Don't worry about any changes you have made before today, I will find them.

**Comment by john-shepherdson** [comment]
@KristinaS4 I have updated the text on the home page, basic search and language pages and inserted images of the rendered pages into the GDoc. Please check. 

If you updated any other pages before today, please let me know.

**Comment by KristinaS4** [comment]
@john-shepherdson I have made some updates to all of the sections before yesterday. I have also made some minor changes today.

**Comment by john-shepherdson** [comment]
I have done my best to find all the changes you made, and implement them.
I have added updated screenshot of all the pages and rerun the advanced search examples and updated the figures.
All links (internal and external) have been checked.

**Comment by john-shepherdson** [comment]
@KristinaS4 I have also updated the About text, and added a screenshot

**Comment by KristinaS4** [comment]
Thank you @john-shepherdson! I think everything looks good, except one mistake in one image that I have indicated in the GDoc. Thank you for checking the links and updating number of results for the Advanced search examples.

**Comment by john-shepherdson** [comment]
@KristinaS4 Now fixed

**Comment by matthew-morris-cessda** [comment]
Reopening as the changes haven't been merged yet
